Filewatcherd-TypeScript/package.json

```diff
@@ -21,7 +21,7 @@
         "package": "./npm-package.sh"
     },
     "dependencies": {
-        "chokidar": "^3.0.2",
+        "chokidar": "^3.3.1",
```

Comment:
After I ran `npm install` on yesterday's filewatcher build, I hit a `TypeError` in chokidar, which I suspect was because you were developing against 3.0.2 but npm installed 3.3.1 when I installed the filewatcher.

https://github.com/eclipse/codewind-vscode/commit/cc45ba8dd4db31da29c0290841eac23de6702982

Do you think you could change chokidar to only pick up micro releases, not minor ones too? IE `~` instead of `^`. in theory the API should not break between minor versions but it did in this case.

Reply:
Agreed and done. I happened to see that PR of yours last night and was wondering: do you by chance still have the error logs? I'm curious if it was similar to Chokidar errors I was seeing while upgrading to 3.3.1...

(@tetchel)

Reply:
IIRC it was `TypeError: listener is not a function`

Reply:
i'll ping you the log